CARTOGRAPHY.md

```diff
@@ -37,3 +37,67 @@ account.
+
+Transparency should be used very carefully with consideration for the color mixing this can result in and the ambiguities possibly caused by it.
+
+It is generally advisable to design color relationships in perceptual color spaces.  Use of internal color functions like `lighten()` and `darken()` should be considered carefully since they are performed in sRGB.
+
+
+### Zoom levels
+
+An important aspect of styles for interactive maps is that they are used for a large number of different map sizes or zoom levels.  This creates special challenges for map design.  A particular problem with this stems from the fact that the Mercator projection is a variable scale projection so the different zoom levels do not represent specific map scales.  This fact should always be considered when making design decisions.  Styling should always work for the whole map and for the whole range of map scales present in the zoom level in question.  This normally requires testing changes in different areas at different latitudes.
+
+A number of rules should generally be followed regarding cross zoom level design:
+
+* Design differences between zoom levels should be small and used with caution.  Map users should be able to read the map without learning to interpret every zoom level separately.  Continuous or small step changes over multiple levels is preferable in comparison to constant styling across multiple levels followed by a huge step.  When increasing dimensions (line widths, text sizes, icon sizes etc.) with higher zoom levels this size increase should not be larger than the scale ratio between the zoom levels, in other words: sizes in which things are drawn should not increase relative to ground units with increasing zoom level.
+* Many map features start being displayed at a certain zoom level and continue being shown, sometimes with changing styling, at subsequent higher levels.  What is highly undesirable though is to show a feature first and then have it vanish again at a higher zoom level.  The only situation where this can be appropriate is in case of labels for features that get very large and where labeling them is therefore no more useful.
+* Decisions at what zoom level to start showing a certain feature type should prominently consider that not actually rendering a large number of elements because of competing labels or icons is often diametrical to the purpose of providing useful mapper feedback, in particular if feature priorities are not based on a meaningful property.
+* Decisions at what zoom level to start showing something are also particularly prone to causing bad mapping decisions (mapping for the renderer).  This should be considered when making choices here.
```

Comment:
In this case considering the population tag would probably help.  But you have to be careful of course so the rendering rules are still comprehensible for mappers.

To be frank - if we right now had the decision to make if and how to render place=city/town/village/hamlet i would probably be against differentiated rendering at all because the wiki lacks a clear definition of the tags and the data is globally quite inconsistent on the matter.  place =city essentially just means here the mapper thought this is a place that should be shown significantly earlier and more prominently than other populated places.